builder.go

```diff
@@ -237,7 +237,7 @@ func (h *hostFunctionBuilder) WithGoFunction(fn api.GoFunction, params, results
 	h.fn = &wasm.HostFunc{
 		ParamTypes:  params,
 		ResultTypes: results,
-		Code:        &wasm.Code{GoFunc: fn},
```

Comment:
hate to ask, as you are on a roll, but do you think this is why the extern cache perf degraded?

Reply:
No, I don't think so.